demo/src/faber_inquirer.ts

```diff
@@ -0,0 +1,133 @@
+import { clear } from 'console'
+import { textSync } from 'figlet'
+import inquirer from 'inquirer'
+
+import { BaseInquirer } from './base_inquirer'
+import { Faber } from './faber'
+import { Listener } from './listener'
+import { Title } from './output_class'
+
+export const runFaber = async () => {
+  clear()
+  console.log(textSync('Faber', { horizontalLayout: 'full' }))
+  const faber = await FaberInquirer.build()
+  faber.processAnswer()
+}
+
+enum PromptOptions {
+  Connection = 'setup connection',
```

Comment:
Maybe we can make it more clear what kind of the flow each side will handle?

E.g. `Create connection invitation` for Alice, and `Receive connection invitation` for Faber. Currently both have `setup connection` which makes it hard to know which agent does what